src/js/veteran-representative/definitions/address.js

```diff
@@ -0,0 +1,160 @@
+import _ from 'lodash/fp';
```

Comment:
I think this file can be deleted if we can move to the common address definition.